### DISCUSSION TITLE
Workspace descriptions for unnamed workspaces, like what is possible in `i3` with `i3wsr`

### DISCUSSION THREAD

**Discussion by JohnDowson** [YaLTeR/niri#900] [Ideas]
I need all help I can get to remember which workspace has which app open.

**Comment by YaLTeR** [comment]
I'm not familiar with this. Do named workspaces not work for this?

**Reply by JohnDowson** [reply]
The idea is that numbered workspaces get dynamic display names based on the windows they contain.

**Reply by YaLTeR** [reply]
Sounds like you could implement it in the bar that you're using? The niri IPC lets you know what windows are on which workspace

**Reply by JohnDowson** [reply]
I could but it'd have to be (partially?) reimplemented for different bars.

**Reply by YaLTeR** [reply]
Well, niri's job in this case is to provide the necessary info so you can implement features that you need
